openlibrary/core/models.py

```diff
@@ -860,22 +860,9 @@ def _get_lists_uncached(self, seed=None, limit=100, offset=0):
 
         return self._site.things(q)
 
-    def new_list(self, name, description, seeds, tags=None):
+    def new_list(self, name, description, tags=None):
```

Comment:
Let's roll back the changes in this file ; it turns out this API already supported creating a list without seeds. We don't want to make it only work without seeds!

Reply:
Sure thing. What method creates a list without requiring a seed?

Reply:
This one :P Specifying seeds as an empty list `[]` works as expected :+1:

Reply:
> This one :P Specifying seeds as an empty list `[]` works as expected 👍

Ok thanks.